agent-install/agent-install.sh

```diff
@@ -2668,7 +2918,9 @@ function loadClusterAgentImage() {
             log_fatal 1 "Can not specify both AGENT_K8S_IMAGE_TAR_FILE and -i (INPUT_FILE_PATH)"
         fi
         if [[ $INPUT_FILE_PATH == css:* ]]; then
-            download_css_file "$INPUT_FILE_PATH/$AGENT_K8S_IMAGE_TAR_FILE"
+            local input_path
+            get_input_file_css_path input_path
+            download_css_file "$input_path/$AGENT_K8S_IMAGE_TAR_FILE"
```

Comment:
Should we add this logic for downloading cronjob image at line: 2997 (https://github.com/open-horizon/anax/blob/63fba0a7a144c0978bfa2f660b1b92d6a5204c38/agent-install/agent-install.sh#L2997)

Reply:
yes, will do.